SPECS/rust/rust.spec

```diff
@@ -73,12 +73,13 @@ mv %{SOURCE7} "$BUILD_CACHE_DIR"
 export CFLAGS="`echo " %{build_cflags} " | sed 's/ -g//'`"
 export CXXFLAGS="`echo " %{build_cxxflags} " | sed 's/ -g//'`"
 
-sh ./configure --prefix=%{_prefix} --enable-extended --tools="cargo"
+sh ./configure --prefix=%{_prefix} --enable-extended --tools="cargo,rustfmt"
 # SUDO_USER=root bypasses a check in the python bootstrap that
 # makes rust refuse to pull sources from the internet
 USER=root SUDO_USER=root %make_build
 
 %check
+ln -s %{_prefix}/src/mariner/BUILD/rustc-1.56.1-src/build/x86_64-unknown-linux-gnu/stage2-tools-bin/rustfmt %{_prefix}/src/mariner/BUILD/rustc-1.56.1-src/build/x86_64-unknown-linux-gnu/stage0/bin/
```

Comment:
Couldn't we use `%{buildroot}` here instead? If not, please reuse `%{version}` so that we do not need to update it every time we change rust version.

Reply:
+1, let's use both `%{buildroot}` and `%{version}`.

Reply:
Let me use `%{version}`.

`╰─$ rpm --eval %{buildroot}
/home/bala/rpmbuild/BUILDROOT/%{NAME}-%{VERSION}-%{RELEASE}.x86_64`
The path should be relative to BUILDROOT and it will be an ugly relative path. Unfortunately there is no `%{build}` macro

And `%{name}` also will not work as it resolves to `rust`. But this package is `rustc`.

Reply:
Ah, I hoped it was a macro for the `build` directory. Understood, thanks for the details!